core/src/templates/components/card/card.json

```diff
@@ -1,7 +1,7 @@
 {
   "card_media_wrapper_modifier_class": "su-aspect-ratio",
-  "card_media_image_src": "https://placeimg.com/2000/1333/any",
```

Comment:
placeimg has retired.

Reply:
placekitten is still alive through https://placekitten.com/ 😸